### PR TITLE
Fix bitwise chiplet error & improve bitwise tests

### PR DESCRIPTION
**Pull request by grjte**
This PR addresses some issues with the modules related to the bitwise chiplet and its tests.

- fixes an error swapping the indices of prev_output and output columns
- minor refactor of consts for consistency (prefix name with `BITWISE_` when indexing globally)
- updates bitwise unit tests in the processor to use constants (this would have caught the error)
- refactors the air unit tests to simplify and reduce the code, now that the prev_output column has been added
- removes an unnecessary unit test from the air tests (irrelevant because output is now aggregated differently)
- adds verification of the previous column to the unit tests of the bitwise execution trace generation in the processor

### CODE REVIEW

File: core/src/lib.rs [hunk 1]
```diff
@@ -7,7 +7,7 @@ extern crate alloc;
 use core::ops::Range;
 
 pub mod chiplets;
-pub use chiplets::{bitwise, hasher};
+pub use chiplets::hasher;
```
Comment (bobbinth) [inline core/src/lib.rs:10]: I'm assuming removing this is left for future PRs, right?

File: core/src/chiplets/mod.rs [hunk 2]
```diff
@@ -24,3 +24,28 @@ pub const HASHER_TRACE_OFFSET: usize = CHIPLETS_OFFSET + NUM_HASHER_SELECTORS;
 pub const BITWISE_TRACE_OFFSET: usize = CHIPLETS_OFFSET + NUM_BITWISE_SELECTORS;
 /// The first column of the memory chiplet.
 pub const MEMORY_TRACE_OFFSET: usize = CHIPLETS_OFFSET + NUM_MEMORY_SELECTORS;
+
+// --- GLOBALLY-INDEXED CHIPLET COLUMN ACCESSORS --------------------------------------------------
+
```
Comment (bobbinth) [inline core/src/chiplets/mod.rs:29]: Not for this PR, but I wonder if we should add diagrams of how the trace looks like here (similar to how we do in other places).

**Review by bobbinth — APPROVED** [review]
Looks good! Thank you! I left a couple of comments inline - but these are for future PRs.

File: core/src/lib.rs (hunk 1 above)
Comment (grjte) [reply core/src/lib.rs:10]: Yes, exactly. I figured I'd do it with the hasher updates.

File: core/src/chiplets/mod.rs (hunk 2 above)
Comment (grjte) [reply core/src/chiplets/mod.rs:29]: Yeah, I think that's a good idea

**Pull request merged**